src/test/CMakeLists.txt

```diff
@@ -356,8 +356,72 @@ target_link_libraries(ceph_multi_stress_watch librados global radostest
 add_executable(ceph_perf_local 
   perf_local.cc
   perf_helper.cc)
+
+# create a tmp file with an empty main()
```

Comment:
maybe we should extract this piece of code into a cmake module in `cmake/modules`, probably named "CheckCompileTargets", maybe a better name? and check the define variables here.